sdk/keyvault/azure-keyvault-securitydomain/tests/test_security_domain.py

```diff
@@ -31,14 +36,16 @@ def test_security_domain_download_and_upload(
         assert status.status.lower() == "inprogress"
         assert result.value
 
+        # Get the transfer key of the secondary HSM, and write it and the security domain to files.
         jwk = str(upload_client.get_transfer_key().transfer_key)
         transfer_key = json.loads(jwk)
         write_transfer_key(transfer_key)
         write_security_domain(result.value)
+        # Wrap the security domain using the certificate{x}.pem keys.
+        wrapped_security_domain = use_wrapping_keys()
 
-        # At this point, use the Azure CLI to encrypt the security domain to prepare for upload
-        # `az keyvault security-domain restore-blob --sd-exchange-key <>-transfer-key.pem --sd-file <>-security-domain.json --sd-wrapping-keys <>-certificate0.key <>-certificate1.key <>-certificate2.key --sd-file-restore-blob <>-restore-blob.json`
-        poller = upload_client.begin_upload(security_domain=result)
```

Comment:
How did this used to work by re-uploading the unwrapped result?

Reply:
It wouldn't work if executed as-is since the result would be unwrapped, so what I would do is set a breakpoint, call the CLI's `restore-blob` command to wrap the security domain, and then reassign the value of `result` before continuing execution. Very much an improvised solution 😅